src/components/Transfer/utils/resolveWidthFromString.ts

```diff
@@ -0,0 +1,3 @@
+export const resolveWidthFromString = (width: string): number => {
+    return +width.substring(0, width.indexOf('p'));
```

Comment:
any string that starts with a number can be parsed with `parseInt` ex. `parseInt("500px") === 500`

strings are iterable in js and share some methods with arrays, if you wish to remove an amount of characters from the end of a string, you can use `.slice(0,-amount)`